.travis.yml

```diff
@@ -30,4 +30,4 @@ script:
     - mkdir build; cd build
     - cmake -D ENABLE_TESTING=ON ../
     - make
-    - ctest
+    - make test
```

Comment:
Add a custom target for calling `ctest` with the `-V` flag to help debugging any failing tests which might be failing exclusively on travis

Reply:
Will do this on its own

Reply:
Actually added this